### ISSUE TITLE
MD002 triggered when frontmatter title is present

### ISSUE THREAD

**Issue by bschlenk** [DavidAnson/vscode-markdownlint#34]
It looks like https://github.com/DavidAnson/markdownlint/issues/19 is closed, which suggests MD002 shouldn't be triggered if a title property is present. My markdown:

```md
---
title: My Markdown File
tags: 
---

Some text.

## First level 2 header.
```

The error I'm seeing:

```
[markdownlint] MD002/first-header-h1: First header should be a top level header [Expected: h1; Actual: h2]
```

**Comment by DavidAnson** [comment]
I suspect you are not setting the `level` parameter to 2 in the config for `MD002`. Could you please check?

For reverence, this related bug has an example that shows your scenario working: https://github.com/DavidAnson/markdownlint/issues/116

**Comment by DavidAnson** [comment]
Please reopen if the suggestion above did not help.
